### DISCUSSION TITLE
add introductions of project architecture in docs

### DISCUSSION THREAD

**Discussion by LoTwT** [vuejs/devtools#192] [General]
## description

Introductions of project architecture in docs may help potential contributors get started and contribute more quickly and easily.

Also, I think the design of RPC communications and bridgeRpc is interesting. I think it's a good topic to share with others. 😊

## expect

- dependencies graph of packages
- introductions of bridgeRpc
- RPC and vite contexts
- introductions of API references in different packages and runtimes
- ...

**Comment by webfansplz** [comment]
Of course, this is in our plan, and we will introduce them in the documentation (when they are more stable).
